gate-web/src/main/groovy/com/netflix/spinnaker/gate/services/DataService.groovy

```diff
@@ -39,14 +39,14 @@ class DataService {
 
   void getStaticData(String id, Map<String, String> filters, OutputStream outputStream) {
     voidCommand("static", {
-      def response = clouddriverServiceSelector.select(null).getStaticData(id, filters)
+      def response = clouddriverServiceSelector.select().getStaticData(id, filters)
```

Comment:
this is interesting where we explicitly didn't want to select in the past but now we do. Probably not an issue, but curious

Reply:
true, I'd say eventually we want every call to go through a common selector for consistent behavior